src/backend/tcop/postgres.c

```diff
@@ -5351,7 +5352,7 @@ curl_finalize(int code, Datum arg __attribute__((unused)))
 		/* we're done with libcurl, so clean it up */
 		curl_global_cleanup();
 		curl_context.hasInited = false;
-		elog(LOG, "finalize the global struct for curl handle context.");
+		elog(DEBUG3, "finalize the global struct for curl handle context.");
```

Comment:
This line for finalize should correspond to line:4649,  "finalize the global curl context for privileges check." So that initialize and finalize appear in pair.